- Enable GPIO and mux

From: Shuotian Cheng <shuche@microsoft.com>

CONFIG_I2C_MUX_PCA954x=m
CONFIG_I2C_GPIO=m
CONFIG_GPIO_PCA953X=m

- Cisco N2200 AC/DC Controllers
CONFIG_SENSORS_CISCO_N2200=m

- LM25066 and TPS40422 drivers
CONFIG_SENSORS_LM25066=m
CONFIG_SENSORS_TPS40422=m

- Unset SERIAL_8250_PNP
CONFIG_SERIAL_8250_PNP is not set
---
 debian/build/build_amd64_none_amd64/.config |   12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
index c63bdf3..1eb1257 100644
--- a/debian/build/build_amd64_none_amd64/.config
+++ b/debian/build/build_amd64_none_amd64/.config
@@ -3050,7 +3050,7 @@ CONFIG_N_GSM=m
 CONFIG_SERIAL_EARLYCON=y
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
-CONFIG_SERIAL_8250_PNP=y
+# CONFIG_SERIAL_8250_PNP is not set
 CONFIG_SERIAL_8250_CONSOLE=y
 CONFIG_SERIAL_8250_DMA=y
 CONFIG_SERIAL_8250_PCI=y
@@ -3190,7 +3190,7 @@ CONFIG_I2C_SCMI=m
 CONFIG_I2C_DESIGNWARE_CORE=m
 CONFIG_I2C_DESIGNWARE_PLATFORM=m
 CONFIG_I2C_DESIGNWARE_PCI=m
-# CONFIG_I2C_GPIO is not set
+CONFIG_I2C_GPIO=m
 CONFIG_I2C_KEMPLD=m
 CONFIG_I2C_OCORES=m
 CONFIG_I2C_PCA_PLATFORM=m
@@ -3304,7 +3304,7 @@ CONFIG_GPIO_SCH=m
 #
 # CONFIG_GPIO_MAX7300 is not set
 # CONFIG_GPIO_MAX732X is not set
-# CONFIG_GPIO_PCA953X is not set
+CONFIG_GPIO_PCA953X=m
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_GPIO_ADP5588 is not set
 
@@ -3484,15 +3484,17 @@ CONFIG_SENSORS_PCF8591=m
 CONFIG_PMBUS=m
 CONFIG_SENSORS_PMBUS=m
 # CONFIG_SENSORS_ADM1275 is not set
-# CONFIG_SENSORS_LM25066 is not set
+CONFIG_SENSORS_LM25066=m
 # CONFIG_SENSORS_LTC2978 is not set
 # CONFIG_SENSORS_MAX16064 is not set
 # CONFIG_SENSORS_MAX34440 is not set
 CONFIG_SENSORS_DNI_DPS460=m
 # CONFIG_SENSORS_MAX8688 is not set
+CONFIG_SENSORS_TPS40422=m
 # CONFIG_SENSORS_UCD9000 is not set
 CONFIG_SENSORS_UCD9200=m
 # CONFIG_SENSORS_ZL6100 is not set
+CONFIG_SENSORS_CISCO_N2200=m
 # CONFIG_SENSORS_SHT15 is not set
 CONFIG_SENSORS_SHT21=m
 # CONFIG_SENSORS_SHTC1 is not set
